src/datajudge/utils.py

```diff
@@ -0,0 +1,48 @@
+from typing import Tuple, Union
+
+from colorama import just_fix_windows_console
+
+just_fix_windows_console()
```

Comment:
Doesn't AnsiColorFormatter already take care of this?

Reply:
Good point. Once this is resolved we can merge! 🚀

Reply:
nice, thanks!